releases/v2.8.3.toml

```diff
@@ -0,0 +1,41 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "registry"
+github_repo = "distribution/distribution"
+
+# previous release
+previous = "v2.8.2"
+
+pre_release = false
+
+preface = """\
+Welcome to the 2.8.3 release of registry!
+
+The 2.8.3 registry release fixes a few bugs and makes a few dependency updates.
+The Go runtime has been bumped to 1.20.8.
+
+See the changelog below for full list of changes.
+
+### Bugfixes
+
+* Do not close HTTP request body in HTTP handler ([#4068](https://github.com/distribution/distribution/pull/4068))
+* Set Content-Type header in registry client ReadFrom ([#4053](https://github.com/distribution/distribution/pull/4053))
+* Do not parse errors as JSON unless Content-Type is set to JSON ([#4054](https://github.com/distribution/distribution/pull/4054))
+* Enable Go build tags ([#4009](https://github.com/distribution/distribution/pull/4009))
```

Comment:
Wondering if we should describe what it fixes (the missing storage drivers); not a hard blocker though

Reply:
Users can navigate to that URL. I thought that was the whole point of including these URLs because they turn into clickable links in the release notes 🤔 (https://github.com/distribution/distribution/releases)

Reply:
Yeah, thought it would be nice to mention the fix for this one, but not a blocker